src/node/utxo_snapshot.h

```diff
@@ -13,7 +13,7 @@
 
 #include <optional>
 
-extern RecursiveMutex cs_main;
+extern RecursiveMutex cs_main; // NOLINT(readability-redundant-declaration)
```

Comment:
https://github.com/bitcoin/bitcoin/pull/26965

Reply:
> #26965

Yeah, it would be nice if #26965 go first.